tests/phpunit/includes/bootstrap.php

```diff
@@ -46,13 +46,83 @@
 	exit( 1 );
 }
 
-// Check that the PHPUnit Polyfills autoloader exists.
-$phpunit_polyfills_autoloader = __DIR__ . '/../../../vendor/yoast/phpunit-polyfills/phpunitpolyfills-autoload.php';
-if ( ! file_exists( $phpunit_polyfills_autoloader ) ) {
-	echo 'Error: You need to run `composer update` before running the tests.' . PHP_EOL;
-	echo 'You can still use a PHPUnit phar to run them, but the dependencies do need to be installed.' . PHP_EOL;
+/*
+ * Load the PHPUnit Polyfills autoloader.
+ *
+ * The PHPUnit Polyfills are a requirement for the WP Test Suite.
+ *
+ * Plugin/theme integration tests can handle this in any of the following ways:
+ * - When using a full WP install: run `composer update` for the WP install prior to running the tests.
+ * - When using a partial WP test suite install:
+ *   - Add a `yoast/phpunit-polyfills:"^1.0.1"` requirement to their own `composer.json` file.
```

Comment:
It seems like it would be easy to forget to update this line when the Polyfills are updated in the future, would that be an issue?
Maybe we could omit the version here, or point to check out the currently supported version in the WP's `composer.json` file or in `$phpunit_polyfills_minimum_version below?`

Reply:
Good point. Not sure why I added the version nr there, I think omitting it is fine. If the version isn't high enough, the end-user will get a notification about that anyway.